### PR TITLE
Update newcommitter.md

### PR DESCRIPTION
**Pull request by cottage14**
Minor text edits for clarity. I note that the final email template talks about SVN but not about Git. Since 90% + of projects now use Git, additional text would be helpful.

### CODE REVIEW

**Review by palejana — APPROVED** [review]

File: source/newcommitter.md
Comment (clr-apache) [inline source/newcommitter.md:98]: If you want to make it clear what the mail list is called, perhaps private@ is a better term than @private.

File: source/newcommitter.md
Comment (clr-apache) [inline source/newcommitter.md:113]: private@

File: source/newcommitter.md
Comment (clr-apache) [inline source/newcommitter.md:134]: private@

File: source/newcommitter.md
Comment (clr-apache) [inline source/newcommitter.md:87]: I have additional edits to the onboarding process that will wait until this PR is accepted.

File: source/newcommitter.md
Comment (clr-apache) [inline source/newcommitter.md:298]: The trailing . might make it hard to simply click on the link unless there is some metadata around it.

File: source/newcommitter.md
Comment (clr-apache) [inline source/newcommitter.md:302]: Forrest is now in the attic and might not be a great example any more...

**Review by clr-apache — CHANGES_REQUESTED** [review]
With the suggested changes, I approve this PR.

**Comment by swapnilmmane** [comment]
Thank you @cottage14 for your work.
As the merge branch, `cottage14:patch-26` for this PR is from a different repository and edits from maintainers were not allowed, I was not able to make updates suggested by @clr-apache.
 
Thus I incorporated these changes at #62 and made the suggested changes from Craig, thank you @clr-apache for your inputs.

File: source/newcommitter.md
Comment (swapnilmmane) [reply source/newcommitter.md:302]: Good point @clr-apache
I have created Jira ticket for it https://issues.apache.org/jira/browse/COMDEV-421
So, it will not block us from merging the all other suggested changes which is incorporated #62

**Comment by rbowen** [comment]
@swapnilmmane Does that mean we need to close this PR?

**Comment by swapnilmmane** [comment]
Hi @rbowen, I will suggest waiting for #62 to be merged. I think, after that this PR will be marked as merged automatically.
At least this similar thing has happened for #61 and #25.
If this PR is not marked as merged after merging #62, then I will close this PR.

**Comment by swapnilmmane** [comment]
Great, merging #62 results in an automatic merging of this PR (#41).

**Pull request merged**
